app/models/post.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: posts
+#
+#  id         :bigint           not null, primary key
+#  body       :text             not null
+#  status     :integer          default("public"), not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  user_id    :bigint
+#
+# Indexes
+#
+#  index_posts_on_user_id  (user_id)
+#
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :pictures, as: :picturable
+  has_many :likes, as: :likeable
+  has_many :post_tags
+  has_many :tags, through: :post_tags
+
+  enum status: %i[public private], _suffix: true
+
+  VALIDATE_MIN_CHARACTERS_BODY = 2
```

Comment:
VALIDATE_MIN_CHARACTERS_BODY starts with a verb. Constant does not to anything. And in your case it does not indicate any action related variable. This constant indicated the amount of minimal characters allowed for post body. So it should be called `BODY_MIN_CHARACTERS`